get.sh

```diff
@@ -87,13 +87,17 @@ getBinaryOpenjdk()
 	fi
 	jarDir=`ls -d */`
 	dirName=${jarDir%?}
-	mv $dirName j2sdk-image
+	if [ "$dirName" != "j2sdk-image" ]; then
+		mv $dirName j2sdk-image
+	else
+		echo "dirName is equal to j2sdk-image, skip moving"
+	fi
 }
 
 getTestKitGen()
 {
 	cd $TESTDIR
-	git clone https://github.com/eclipse/openj9.git
+	git clone --single-branch --depth 2 https://github.com/eclipse/openj9.git
```

Comment:
To my understand if using --depth option --single-branch would be default option?

```
 git clone --depth 1 https://github.com/eclipse/openj9.git 

```
Would do same function?